apps/daimo-mobile/src/view/shared/OfflineHeader.tsx

```diff
@@ -12,9 +13,13 @@ import { useNetworkState } from "../../sync/networkState";
 export function OfflineHeader({
   dontTakeUpSpace,
   offlineExtraMarginBottom,
+  onPress,
+  title,
 }: {
   dontTakeUpSpace?: boolean;
   offlineExtraMarginBottom?: number;
+  onPress?(): void;
+  title?: string;
```

Comment:
I don't think we need these. The pull to refresh and tap can be separate.

I recommend following:
* On OfflineHeader tap (when offline), call `resync` and change the text to Retrying...
* This can be done locally to OfflineHeader

I think this will make it work on all tabs (vs just HomeTab).